guides/04_additional-features/08_file-access.md

```diff
@@ -0,0 +1,64 @@
+# File Access
```

Comment:
Suggest including the word security here to make it clear what this section is around, perhaps:
```suggestion
# Security and File Access
```